src/main/webapp/i18n/en/aboutUs.json

```
@@ -1,34 +1,103 @@
+                "details": [
+                    "<b>Multiple diagram types:</b> Artemis supports 7 UML diagram types (e.g. class diagrams) and 4 additional diagrams (e.g. flow charts).",
+                    "<b>Easy to use:</b> Students can create models easily using drag and drop, they can explain the models using additional text.",
+                    "<b>Integrated feedback:</b> Reviews can provide feedback and points directly next to the model elements."
+                ]
+            },
+            "text": {
+                "text": "<a href=\"https://docs.artemis.ase.in.tum.de/user/exercises/textual/\">Text exercises</a> with manual, semi-automatic assessment based on supervised machine learning and natural language processing (NLP) using <a href=\"https://github.com/ls1intum/Athena\">Athena</a>.",
+                "details": [
+                    "<b>Integrated feedback:</b> Reviews can provide feedback and points directly next to the text segments.",
+                    "<b>Language detection:</b> Artemis detects the language of the submission and shows the word and character count."
+                ]
+            },
+            "fileUpload": {
+                "text": "<a href=\"https://docs.artemis.ase.in.tum.de/user/exercises/file-upload/\">File upload exercises</a> with manual assessment",
+                "details": []
             }
         },
         "exam": {
-            "title": "Artemis supports the following features for exams",
-            "detail": {
-                "supportedExerciseTypes": "All exercises from above are also available for exams",
-                "randomSelectionOfVariants": "Multiple variants per exercise can be created and randomly selected for every student",
-
-                "fullFeatureOverview": {
-                    "description": "Complete overview of all features:",
-                    "studentLink": "Overview for students",
-                    "instructorLink": "Overview for instructors"
-                },
-                "instructions": {
-                    "description": "Guide for the exam feature:",
-                    "studentLink": "Instructions for students",
-                    "instructorLink": "Instructions for instructors"
-                }
-            }
+            "text": "<b>Exam mode:</b> Instructors can create online exams with exercise variants, integrated plagiarism checks, test runs and student reviews. You can find more information on <a href=\"https://artemis.ase.in.tum.de/features/students\">Exam mode student features</a> and on <a href=\"https://artemis.ase.in.tum.de/features/instructors\">Exam mode instructor features</a>.",
+            "details": ["All exercises from above are also available for exams.", "Multiple variants per exercise can be created and randomly selected for every student."]
```

```suggestion
            "details": [
                "All mentioned exercise types are also available for exams.",
                "Multiple variants per exercise can be created and randomly selected for every student."
            ]
```

---

This can't be changed (autoformat)

---

```suggestion
            "details": ["All mentioned exercise types are also available for exams.", "Multiple variants per exercise can be created and randomly selected for every student."]
```

Okay. The other part of the suggestion should still be possible, though.

---

Yeah i missed that :D thought that was only the layout :D solved :+1: